src/session.ts

```diff
@@ -191,6 +195,9 @@ export class SessionManager implements Middleware {
 
         if (this.sessionSettings.integratedConsole.suppressStartupBanner) {
             this.editorServicesArgs += "-StartupBanner '' ";
+        } else {
+            const startupBanner = `=====> ${this.HostName} Integrated Console v${this.HostVersion} <=====${os.EOL}`;
```

Comment:
I'm not sure if `os.EOL` is required here or if I can just use `\n`. It works on Windows. Not sure about macOS though.

Reply:
It should work on macOS though os.EOL is probably good practice.